explorer/src/components/account/MetaplexNFTAttributesCard.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { NFTData } from "providers/accounts";
+import { LoadingCard } from "components/common/LoadingCard";
+import { ErrorCard } from "components/common/ErrorCard";
+
+interface Attribute {
+  trait_type: string;
+  value: string;
+}
+
+export function MetaplexNFTAttributesCard({ nftData }: { nftData: NFTData }) {
+  const [attributes, setAttributes] = React.useState<Attribute[]>([]);
+  const [status, setStatus] = React.useState<"loading" | "success" | "error">(
+    "loading"
+  );
+
+  async function fetchMetadataAttributes() {
+    try {
+      const response = await fetch(nftData.metadata.data.uri);
+      const metadata = await response.json();
+
+      // Verify if the attributes value is an array
+      if (Array.isArray(metadata.attributes)) {
+        // Filter attributes to keep objects matching schema
+        const filteredAttributes = metadata.attributes.filter(
+          (attribute: any) => {
+            return (
+              typeof attribute === "object" &&
+              typeof attribute.trait_type === "string" &&
+              typeof attribute.value === "string"
```

Comment:
Can this be a number?

Reply:
You're right, according to Metaplex documentation:
```
value can be a string or a number.
```

https://docs.metaplex.com/token-metadata/Versions/v1.0.0/nft-standard